src/lib/openapi/spec/search-events-schema.ts

```diff
@@ -0,0 +1,29 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const searchEventsSchema = {
+    $id: '#/components/schemas/searchEventsSchema',
+    type: 'object',
+    properties: {
+        type: {
+            type: 'string',
+        },
+        project: {
+            type: 'string',
+        },
+        feature: {
+            type: 'string',
+        },
+        query: {
```

Comment:
It could help to add some more description here. As I understand the rest for the code you can combine a query with other filters (like project, type, feature toggle name). Would be good if this is briefly described to provide some more details for the api user.